### PR TITLE
update to julia 1.8.1

### PR DESCRIPTION
**Pull request by trontrytel**

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]

File: .github/workflows/CodeCov.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Julia
       uses: julia-actions/setup-julia@latest
       with:
-        version: 1.8.0
+        version: 1.8.1
```
Comment (simonbyrne) [inline .github/workflows/CodeCov.yml:22]:
```suggestion
        version: 1.8
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.8.0'
+          - '1.8.1'
```
Comment (simonbyrne) [inline .github/workflows/ci.yml:19]:
```suggestion
          - '1.8'
```

File: .github/workflows/julia_formatter.yml [hunk 3]
```diff
@@ -31,7 +31,7 @@ jobs:
     - uses: julia-actions/setup-julia@latest
       if: steps.filter.outputs.julia_file_change == 'true'
       with:
-        version: 1.8.0
+        version: 1.8.1
```
Comment (simonbyrne) [inline .github/workflows/julia_formatter.yml:34]:
```suggestion
        version: '1.8'
```

**Review by simonbyrne — COMMENTED** [review]
In GitHub Actions we can just use 1.8

**Pull request merged**